commune_app/tests/test_chores.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+from commune_app import models
+from commune_app.models import User, Chore
```

Comment:
I think you can do it like:
```suggestion
from commune_app.models import User, Chore
```

Reply:
done